tinyagentos/restart_orchestrator.py

```
@@ -243,49 +243,204 @@ async def apply_pending_restart_check(app_state) -> None:
+#   - agents whose containers boot slower than the controller failed the single
+#     resume attempt and stayed paused silently.
+
+_RESUME_RETRY_INTERVAL_S = 30
+_RESUME_RETRY_WINDOW_S = 600
+
+
+def _load_or_synthesize_note(note_path: Path) -> dict:
+    if note_path.exists():
+        try:
+            return json.loads(note_path.read_text())
+        except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)